certora/confs/ReentrancyOptimism.conf

```diff
@@ -0,0 +1,19 @@
+{
+  "files": [
+    "src/MorphoTokenOptimism.sol",
+  ],
+  "parametric_contracts" : ["MorphoTokenOptimism"],
+  "solc": "solc-0.8.27",
+  "verify":"MorphoTokenOptimism:certora/specs/Reentrancy.spec",
+  "rule_sanity": "basic",
+  "server": "production",
+  "packages": [
+        "@openzeppelin/contracts/=lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts"
+  ],
+  "prover_args": [
+        "-enableStorageSplitting false"
+    ],
+  "loop_iter" : "3",
+  "optimistic_loop" : true,
```

Comment:
There are no loop in the contract right ? So it would mean we do not need to have this setting. It is also "more sound" to not have it, or rather safer

Reply:
Oh but there are, apparently. See this [run](https://prover.certora.com/output/129828/814c93bcaaec433fa7c8d1e94ba75d49?anonymousKey=ff21adadc17117555e3260656ad32660b814ff02), the functions `permit`, `initialize` and `delegateWithSig` seem to be using loops.

Reply:
I see, it seems like it's not a loop in the solidity code but in the generated bytecode. It seems to be only for retrieving/setting the EIP712 name and version strings. I'm not sure what's the best here, maybe we could manage to summarize it, but maybe it will become too ugly to do

Reply:
I believe that this situation appears in most verifications. Perhaps, acknowledging it in the README is the best to avoid _possible_ impact on other setups?

Reply:
Yes, let's go for that solution in the short term. Long term I think we should ask the Certora team for better string support. I'm adding it to the meeting notes

Reply:
Implemented in https://github.com/morpho-org/morpho-token-upgradeable/pull/90/commits/61d555bc6772a43f7c4e0eb27de11167599697eb.